mlapi.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+import scipy
+import joblib
+from sklearn.preprocessing import RobustScaler
+from flask import Flask, request, jsonify
+
+mlapi=Flask(__name__)
+
+@mlapi.route("/get_question/<int:cnt>",methods=["GET"])
+def recommend_question(cnt):
+    og_dat1=pd.read_csv(r"og_dataset.csv")
+    prob_dat=pd.DataFrame(scipy.sparse.load_npz(r"prob_table.npz").toarray(),columns=pd.read_csv(r"prob_table_col_labels.csv",sep=',')['0'])
+    prob_dat=prob_dat.sort_values(['Probability']).reset_index(drop=True)
+    recomm=og_dat1[og_dat1['Cluster_label']==prob_dat.iloc[cnt,0]][['title','difficulty','related_topics']].to_json(orient='records')
```

Comment:
Why am I only being recommended a title? Where is a specific question ID/question/description? is there no way to refer to these on the OG website, either by ID or a link?

Reply:
i can return the ID too. sorry, missed that one